convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -164,12 +179,28 @@ def run(self):
                         " environment variable.  Please switch to 'CONVERT2RHEL_ALLOW_OLDER_VERSION'"
                         " instead."
                     )
-
+                    self.add_message(
+                        level="WARNING",
+                        id="DEPRECATED_ENVIRONMENT_VARIABLE",
+                        title="Deprecated environment variable detected",
+                        diagnosis="You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_VERSION'",
+                        remediation="Please switch to the 'CONVERT2RHEL_ALLOW_OLDER_VERSION' environment variable instead",
+                    )
                 logger.warning(
                     "You are currently running %s and the latest version of Convert2RHEL is %s.\n"
                     "'CONVERT2RHEL_ALLOW_OLDER_VERSION' environment variable detected, continuing conversion"
                     % (installed_convert2rhel_version, latest_available_version[1])
                 )
+                self.add_message(
+                    level="WARNING",
+                    id="ALLOW_OLDER_VERSION_ENVIRONMENT_VARIABLE",
+                    title="Outdated convert2rhel version detected",
+                    description=(
+                        "You are currently running %s and the latest version of Convert2RHEL is %s.\n"
+                        "'CONVERT2RHEL_ALLOW_OLDER_VERSION' environment variable detected, continuing conversion"
+                        % (installed_convert2rhel_version, latest_available_version[1])
+                    ),
```

Comment:
Maybe this description should be moved to a variable to not mismatch at some point from the warning logger?